packages/node_modules/nav-frontend-lukknapp/src/lukknapp.js

```diff
@@ -29,11 +29,12 @@ class Lukknapp extends Component { // eslint-disable-line react/prefer-stateless
     }
 
     render() {
-        const { children, bla, hvit, overstHjorne, className, ...props } = this.props;
+        const { children, bla, hvit, overstHjorne, className, ariaLabel, ...props } = this.props;
         return (
             <button
                 ref={(buttonRef) => { this.buttonRef = buttonRef; }}
                 className={cls(bla, hvit, overstHjorne, className)}
+                aria-label={ariaLabel}
```

Comment:
Her tipper jeg at vi må bruke `ariaLabel` istedetfor `aria-label` for å slippe warning i console? Ellers ser dette bra ut 👍

Reply:
Tror ikke det. Om jeg ikke tar feil så skrives alle aria-attributter med bindestrek fremfor camelcase i react. 

Ref. 
https://reactjs.org/docs/accessibility.html

Reply:
Ah, ok. Da godkjennes denne fra meg.